tmc-langs-rust/src/test/resources/testCheat/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+
+pub fn mul_xor_add(a: u64, b: u64) -> u64 {
+    a + b
+}
+
+#[test]
+fn it_shall_work() {
+
```

Comment:
Is this function block empty for purpose? I do not know Rust but it seems weird.